src/Illuminate/Broadcasting/Broadcasters/ReboundBroadcaster.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Illuminate\Broadcasting\Broadcasters;
+
+use Illuminate\Contracts\Broadcasting\Broadcaster;
+use Illuminate\Contracts\Redis\Database as RedisDatabase;
+
+class ReboundBroadcaster extends AbstractBroadcaster implements Broadcaster
+{
+    /**
+     * The Redis instance.
+     *
+     * @var \Illuminate\Contracts\Redis\Database
+     */
+    protected $redis;
+
+    /**
+     * The Redis connection to use for broadcasting.
+     *
+     * @var string
+     */
+    protected $connection;
+
+    /**
+     * Create a new broadcaster instance.
+     *
+     * @param  \Illuminate\Contracts\Redis\Database  $redis
+     * @param  string  $connection
+     * @return void
+     */
+    public function __construct(RedisDatabase $redis, $connection = null)
+    {
+        $this->redis = $redis;
+        $this->connection = $connection;
+    }
+
+    /**
+     * Return the valid Rebound authentication response.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  mixed  $result
+     * @return mixed
+     */
+    public function validAuthenticationResponse($request, $result)
+    {
+        return ['status' => 'success', 'user_id' => $request->user()->id, 'user_info' => $request->user()];
```

Comment:
I would prefer to use this in the BroadcastServericeProvider like in @taylorotwell's demo and allow the developer to choose what data is presented in the auth response. In different situations, the user model may contain information that isn't necessary and you may want to pass along additional data. I believe that is what the `$result` variable was intended for.